### DISCUSSION TITLE
My Analog TV Preset

### DISCUSSION THREAD

**Discussion by TheGithubberpng** [ntsc-rs/ntsc-rs#191] [Presets]
[analogtv.json](https://github.com/user-attachments/files/17171531/analogtv.json)
use it

Here's some examples
![analogtvpresetpreview](https://github.com/user-attachments/assets/5c6e7603-d6b5-42db-9394-198e9f51e532)
![colorbars](https://github.com/user-attachments/assets/5831626e-eee6-4059-9d65-4af807d03013)

**Comment by Degamisu** [comment]
Oh wow! Very well done @TheGithubberpng! That's definitely a really close and accurate approach!
